3_rest_requests_multi_model.ipynb

```diff
@@ -97,6 +97,74 @@
     "else:\n",
     "    print('not fraud')"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5697c2ff",
+   "metadata": {},
+   "source": [
+    "## Example 1: user buys a coffee\n",
+    "\n",
+    "In this example, the user is buying a coffee. The parameters given to the model are:\n",
+    "* same location as the last transaction (distance=0)\n",
+    "* same median price (ratio_to_median=1)\n",
+    "* using pin number (pin=1)\n",
+    "* using chip (chip=1)\n",
+    "* no online transaction (online=0)"
+   ]
```

Comment:
some recommended text changes shown here in bold:
* same median price **as the last transaction**
* using **a** pin number
* using the **credit card** chip 
 * **not an** online transaction